moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -155,22 +150,22 @@ bool MoveItCpp::loadPlanningSceneMonitor(const PlanningSceneMonitorOptions& opti
 
 bool MoveItCpp::loadPlanningPipelines(const PlanningPipelineOptions& options)
 {
-  ros::NodeHandle node_handle(options.parent_namespace.empty() ? "~" : options.parent_namespace);
+  // ros::NodeHandle node_handle(options.parent_namespace.empty() ? "~" : options.parent_namespace);
```

Comment:
delete

Reply:
Added a TODO to use the parent namespace for parameter lookup (I fix this in a follow-up PR with additional parameter refactoring)